### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by Aleirbag04**

### CODE REVIEW

File: src/components/screens/LoginScreen.tsx [hunk 1]
```diff
@@ -39,37 +35,52 @@ const LoginScreen = () => {
...
+
+          <Link
             onPress={() => {
               navigation.navigate('Signup');
             }}
-            style={styles.buttonOutline}
           >
-            <Text style={styles.buttonTextOutline}>Create Account</Text>
-          </TouchableOpacity>
+            Create an Account
```
Comment (jatinparmar96) [inline src/components/screens/LoginScreen.tsx:82]: Translations Please 😅

File: src/components/screens/createProfile/IDVerificationScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    });
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text variant="screenTitle">ID Verification</Text>
+      <View style={{ marginHorizontal: 20 }}>
+        <Text>
+          {' '}
+          We need to determine if the identity document is authentic and belongs
```
Comment (jatinparmar96) [inline src/components/screens/createProfile/IDVerificationScreen.tsx:66]: Here too

File: src/components/screens/createProfile/StepOneForm.tsx [hunk 3]
```diff
@@ -134,100 +140,119 @@ export const StepOneForm: React.FC = () => {
...
+        <Text style={styles.title} color="green">
+          Your primary information
+        </Text>
+        <Text
+          textAlign="center"
+          marginLeft="50px"
+          marginRight="50px"
+          marginBottom="21px"
+        >
           Primary information is important to keep our community safe, and this
```
Comment (jatinparmar96) [inline src/components/screens/createProfile/StepOneForm.tsx:153]: Same here

**Review by jatinparmar96 — CHANGES_REQUESTED** [review]

**Review by ernestoguevara92 — APPROVED** [review]

File: src/components/screens/createProfile/IDVerificationScreen.tsx (hunk 2 above)
Comment (Aleirbag04) [reply src/components/screens/createProfile/IDVerificationScreen.tsx]: done

File: src/components/screens/createProfile/StepOneForm.tsx (hunk 3 above)
Comment (Aleirbag04) [reply src/components/screens/createProfile/StepOneForm.tsx]: done

File: src/components/screens/LoginScreen.tsx (hunk 1 above)
Comment (Aleirbag04) [reply src/components/screens/LoginScreen.tsx:82]: done

**Review by jatinparmar96 — APPROVED** [review]

**Pull request merged**